### PR TITLE
Updated release-process.md

### PR DESCRIPTION
**Pull request by amyblais**
Added one task to the checklist - needs a PM confirmation that I put it in the right spot.

### CODE REVIEW

File: source/process/release-process.md [hunk 1]
```diff
@@ -224,6 +224,7 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Review any changes made to install guides, and test if necessary
     - Run loadtests against the final release build to confirm there are no performance issues
     - Review [Security Policies](https://docs.mattermost.com/process/security.html) page is updated
```
Comment (lindy65) [inline source/process/release-process.md]: This wasn't part of AB's updates but wondering whether this should read:
1. Review changes made to [Security Policies](https://docs.mattermost.com/process/security.html) page
or
2. Ensure [Security Policies](https://docs.mattermost.com/process/security.html) page has been updated

File: source/process/release-process.md [hunk 2]
```diff
@@ -224,6 +224,7 @@ The final release is cut. If an urgent and important issue needs to be addressed
...
+    - Update dependancies after release branch is cut
```
Comment (jasonblais) [inline source/process/release-process.md]: Actually, at T-0 looks like there's an item

```
Check if any libraries need to be updated in all repositories, including `mattermost-server`, `mattermost-webapp`, `desktop`, `mattermost-mobile` and `mattermost-redux`
```

but I think moving it to T-2 makes sense, and calling it dependancies. @amyblais Maybe we can just tweak to say

```
Update dependancies after release branch is cut in `mattermost-server`, `mattermost-webapp`, `desktop`, `mattermost-mobile` and `mattermost-redux`
```

File: source/process/release-process.md (hunk 1 above)
Comment (jasonblais) [reply source/process/release-process.md]: @lindy65 I'd go with two, agree

**Comment by jasonblais** [comment]
Thanks, @amyblais also removed the dependancy update from T-0 to avoid duplicate tasks

**Pull request merged**